services/brig/test/integration/Main.hs

```diff
@@ -76,6 +80,27 @@ runTests iConf bConf = do
   where
     mkRequest (Endpoint h p) = host (encodeUtf8 h) . port p
 
+    mkLocalAWSEnv :: Logger.Logger -> Opts.AWSOpts -> Manager -> IO (Maybe AWS.Env)
+    mkLocalAWSEnv l o m
+        | "amazonaws.com" `isSuffixOf` (decodeLatin1 ((Opts.sesEndpoint o)^.awsHost)) = return Nothing
```

Comment:
We only really need an AWS env for SES but it's easier to create full `AWSOpts` so that we can reuse code from `Brig.AWS`